LabExT/View/MovementWizard/MovementWizardController.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+
+import importlib
+import sys
+from typing import Type
+
+from LabExT.Movement.Stage import Stage, StageError
+from LabExT.View.MovementWizard.MovementWizardView import MovementWizardView
+from LabExT.Movement.MoverNew import MoverError, MoverNew
+
+
+class MovementWizardController:
+    def __init__(self, experiment_manager, mover, parent=None) -> None:
+        self.experiment_manager = experiment_manager
+        self.mover: Type[MoverNew] = mover
+        self.view = MovementWizardView(
+            parent, self, self.mover) if parent else None
+
+    def save(
+        self,
+        stage_assignment: dict,
+        speed_xy: float = MoverNew.DEFAULT_SPEED_XY,
+        speed_z: float = MoverNew.DEFAULT_SPEED_Z,
+        acceleration_xy: float = MoverNew.DEFAULT_ACCELERATION_XY,
+    ):
+        """
+        Saves mover configuration.
+
+        Throws ValueError if speed or acceleration is not in the valid range.
+        Throws MoverError and/or StageError if stage assignment or stage connection did not work.
+        """
+        # Tell mover to use the assigned stage. Can throw MoverError,
+        # StageError and ValueError
+        for stage, assignment in stage_assignment.items():
+            try:
+                orientation, position = assignment
+                calibration = self.mover.add_stage_calibration(
+                    stage, orientation, position)
+                calibration.connect_to_stage()
+            except (ValueError, MoverError, StageError) as e:
+                self.mover.reset()
+                raise e
+
+        # Setting speed and acceleration values. Can throw StageError and
+        # ValueError.
+        if self.mover.has_connected_stages:
+            self.mover.speed_xy = speed_xy
+            self.mover.speed_z = speed_z
+            self.mover.acceleration_xy = acceleration_xy
+
+        # Refresh Context Menu
+        if self.experiment_manager:
+            self.experiment_manager.main_window.refresh_context_menu()
```

Comment:
Now I merged the PR #42 too early: there is already a feature to update the availability of the main-window check-boxes depending on the state of LabExT submodules. Its here: 

https://github.com/LabExT/LabExT/blob/e3e5d2d4c52a204c044f47223152f1d6a22bed7f/LabExT/View/MainWindow/MainWindowModel.py#L152

Maybe, you can add another PR to unify the functionality of `submodule_status_updated` and `refresh_context_menu` into something that tells the main GUI to "reload status from all submodules and adapt your GUI widgets accordingly"?

Reply:
Good point, I did not see this method. I can combine them in a second PR.